src/routes/cube/api.js

```diff
@@ -327,23 +327,6 @@ router.post(
   }),
 );
 
-router.get(
```

Comment:
Found no usages of "getimage" (whole world) or "cube/getimage". Found when backtracking from getMostReasonable usages